src/classes/FORM_ServiceGiftEntry_TEST.cls

```diff
@@ -206,6 +206,81 @@ public with sharing class FORM_ServiceGiftEntry_TEST {
             'New template names should return true from `FORM_ServiceGiftEntry.checkNameUniqueness`');
     }
 
+    @isTest
+    static void shouldRetrieveTemplateWithCRUDErrors() {
+        UTIL_PermissionsMock utilPermissionsMockInstance = new UTIL_PermissionsMock(false, 'CRUD');
```

Comment:
Use the constant from the class, instead of hard-coding 'CRUD'.

Reply:
fixed. no longer need 'CRUD' in this class as its being used in `FORM_PermissionsValidator_TEST` for more robust testing there.